src/app/acorns/acorns.html

```diff
@@ -3,7 +3,7 @@
 <kudos-navbar></kudos-navbar>
 
 <div class="container">
-    <div class="panel panel-body">
+   <!-- <div class="panel panel-body">
```

Comment:
There should never be commented code in releasable code, be it javascript, css or html